std/os/windows.zig

```diff
@@ -756,11 +756,23 @@ pub fn sliceToPrefixedSuffixedFileW(s: []const u8, comptime suffix: []const u16)
     return result;
 }
 
+inline fn MAKELANGID(p: c_ushort, s: c_ushort) LANGID {
+    return (s << 10) | p;
+}
+
 /// Call this when you made a windows DLL call or something that does SetLastError
 /// and you get an unexpected error.
 pub fn unexpectedError(err: DWORD) std.os.UnexpectedError {
     if (std.os.unexpected_error_tracing) {
-        std.debug.warn("unexpected GetLastError(): {}\n", err);
+        // 614 is the length of the longest windows error desciption
```

Comment:
Is this documented somewhere?

Reply:
no, this was based on the error messages in the comments in `std/os/windows/error.zig` which match those in the docs, but on another inspection the errors zig defines are far from complete. zig only has error codes 0-3999 with codes 4000-15999 missing.